etc/protractorConfSauce.js

```diff
@@ -1,45 +1,20 @@
-var exec = require("sync-exec");
-var fs = require("fs");
-var ini = require("ini");
 var pr = process.env.TRAVIS_PULL_REQUEST;
+var souceUser = process.env.SAUCE_USERNAME || "liqd";
+var sauceKey = process.env.SAUCE_ACCESS_KEY;
 var name = ((pr === "false") ? "" : "#" + pr + " ") + process.env.TRAVIS_COMMIT;
 
-exports.config = {
-    suites: {
-        // FIXME: mercator tests fail on travis
-        //current: "../src/current/current/tests/acceptance/*Spec.js",
-        core: "../src/adhocracy_frontend/adhocracy_frontend/tests/acceptance/*Spec.js"
-    },
-    baseUrl: "http://localhost:9090",
-    sauceUser: "liqd",
-    sauceKey: "77600374-1617-4d7b-b1b6-9fd82ddfe89c",
+var common = require("./protractorCommon.js");
+
+var local = {
+    sauceUser: souceUser,
+    sauceKey: sauceKey,
 
     capabilities: {
         "browserName": "chrome",
         "tunnel-identifier": process.env.TRAVIS_JOB_NUMBER,
         "build": process.env.TRAVIS_BUILD_NUMBER,
         "name": name
     },
-    beforeLaunch: function() {
-        exec("bin/supervisord");
-        exec("bin/supervisorctl restart adhocracy_test:test_zeo test_backend_with_ws adhocracy_test:test_autobahn adhocracy_test:test_frontend");
-        exec("src/current/current/tests/acceptance/setup_test.sh");
-    },
-    afterLaunch: function() {
-        exec("bin/supervisorctl stop adhocracy_test:test_zeo test_backend_with_ws adhocracy_test:test_autobahn adhocracy_test:test_frontend");
-        exec("rm -rf var/test_zeodata/Data.fs* var/test_zeodata/blobs");
-    },
-    onPrepare: function() {
-        var getMailQueuePath = function() {
-            var testConf = ini.parse(fs.readFileSync("etc/test_with_ws.ini", "utf-8"));
-            return testConf["app:main"]["mail.queue_path"]
-                   .replace("%(here)s", process.cwd() + "/etc");
-        };
-
-        browser.params.mail = {
-            queue_path: getMailQueuePath()
-        }
-    },
     allScriptsTimeout: 21000,
     getPageTimeout: 20000,
     jasmineNodeOpts: {
```

Comment:
Well those values are not determined in any way, I just copied them from the old config.

> both timeouts seem a bit low

They are high enough to get this working with travis reliably (as proven by this pull-requests travis run and others). As always with timeouts it is unclear what the reasoning should be here. If we are willing to extend them to what ever value just to get the tests somehow working, than we could directly configure them to INT.max. If we argue that in an acceptance test loading a page should at maximum take X seconds, otherwise this is no acceptable behaviour for the user than the values are probably already to high. I guess we all agree that inital page load time on almost any internet connection should not exceed 20 seconds. And also after 21 seconds the application should be in a state where everything is loaded.

Obviously those values also depend a bit on connection between the executing host and saucelabs.

> do we need this allScriptsTimeout at all?

I haven't tested it. But this is not completely unrelated to the other request. If the connection to the backend/frontend webservers is slow it will also take longer for the app to finish initial http requests (see [protractor-doc](https://github.com/angular/protractor/blob/master/docs/timeouts.md))